_posts/2024-01-02-resource-recommendations.html

```diff
@@ -0,0 +1,200 @@
+---
+author:
+duration: 4 Min Read
+excerpt: These resources have also formed and shaped Upstream. We’ve allowed these books to influence how we think and approach ministry.
+image: articles/feature-book-shelf.jpg
+image_alt: A shelf of books.
+layout: article
+nav_state: transparent-dark
+permalink: /articles/resource-recommendations/
+title: Resource Recommendations
+---
+
+<h2>Learn With Us</h2>
+
+<p>
+  There is an ocean of resources about missions, technology, faith and work,
+  theology…the list of topics goes on and on. If you’re like us, life is busy
+  and you can’t read everything. We’ve curated a shortlist that we hope
+  illuminates a trusted pathway through the sea of options. Or at least it can
+  serve as a good starting point.
+</p>
+
+<p>
+  These resources have also formed and shaped Upstream. We’ve allowed these
+  books to influence how we think and approach ministry. In this sense, the list
+  also gives a bit of a behind the scenes look into who we are and what we
+  value.
+</p>
+
+<div class="row justify-content-between align-items-center">
+  <div class="start-image-box order-1 col-12 col-md-5 text-center">
+    <img src="{{ "/assets/images/articles/resource_the-five-disfunctions.jpeg" |
```

Comment:
The image width should be the same as the article body text.

From design:
![Screenshot 2024-04-17 at 9 39 35 AM](https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/627c24a3-30ca-4536-8b24-5750693ac8b1)

vs:
<img width="885" alt="Screenshot 2024-04-17 at 9 40 18 AM" src="https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/490860c6-e828-4ad3-8f84-9be0e0fe9b3f">

Reply:
I now realize these comments are under the wrong image, in the wrong file (this adjustment will need to be in _layouts/article.html, line 17).

This change would affect all other articles, so hopefully this is an easy fix across the board!